cucumber/api/features/step_definitions/logs_steps.rb

```diff
@@ -1,17 +1,13 @@
 # frozen_string_literal: true
 
 Given(/^I save the log data from bookmark$/) do
-  lines_amount_in_bookmark()
-end
-
-Given(/^I save the log data from bookmark "([^"]*)"$/) do |bookmark|
-  lines_amount_in_bookmark(bookmark)
+  save_num_log_lines()
```

Comment:
To make the language consistent with our changes in the helpers file, let's change:
```
Given(/^I save the log data from bookmark$/) do
```
to
```
Given(/^I save my place in the log file$/) do
```